packages/dds/tree/src/test/shared-tree-core/utils.ts

```diff
@@ -92,6 +92,20 @@ export class TestSharedTreeCore extends SharedTreeCore<DefaultEditBuilder, Defau
 		);
 		this.changeFamily = changeFamily;
 
+		this.transaction = new SquashingTransactionStack(
```

Comment:
Move initialization to where order is more clear/deterministic and can access members of "this" in more cases. Kernel refactor requires this change to access mintRevisionTag.